src/Tribe/iCal.php

```diff
@@ -439,8 +439,8 @@ protected function get_timezones( $events = [] ) {
 			$timezone = $row['timezone'];
 
 			$ordered = [
-				'start' => array_column( $row['events'], 'start_year' ),
-				'end' => array_column( $row['events'], 'end_year' ),
+				'start' => wp_list_pluck( $row['events'], 'start_year' ),
+				'end' => wp_list_pluck( $row['events'], 'end_year' ),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 3 space(s) between "'end'" and double arrow, but found 1.

<sub>hash: 8a02e61bd79780f4b8cb9a4ef5b80f0f</sub>